Automattic/wp-calypso
# Add Jetpack Cloud Settings Host Credentials Status

#### Changes proposed in this Pull Request

**Motivation**: Add component for top status durning new host-specific credential flow

* Add `<ConnectionStatus />` component to new credentials settings page
* A couple file org changes

![Untitled 10](https://user-images.githubusercontent.com/2810519/92407241-dda9d900-f0ee-11ea-88ee-9dfaf9f9763c.png)


#### Testing instructions

1. boot `env ENABLE_FEATURES=jetpack/server-credentials-advanced-flow yarn run start-jetpack-cloud`
2. verify the `settings` page now matches the "after" screenshot above
